src/test/java/com/netflix/simianarmy/aws/conformity/TestASGOwnerEmailTag.java

```diff
@@ -0,0 +1,80 @@
+// CHECKSTYLE IGNORE Javadoc
+package com.netflix.simianarmy.aws.conformity;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.Lists;
+import com.amazonaws.services.autoscaling.model.AutoScalingGroup;
+import com.amazonaws.services.autoscaling.model.TagDescription;
+import com.netflix.simianarmy.conformity.Cluster;
+import com.netflix.simianarmy.client.aws.AWSClient;
+
+import junit.framework.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.LinkedList;
+
+public class TestASGOwnerEmailTag {
+    private static final String ASG1 = "asg1";
+    private static final String ASG2 = "asg2";
+    private static final String INSTANCE_ID = "i-01234567890";
+    private static final String OWNER_TAG_KEY = "owner";
+    private static final String OWNER_TAG_VALUE = "tyler@paperstreet.com";
+    private static final String REGION = "eu-west-1";
+
+    @Test
+    public void testForOwnerTag() {
+        List<AutoScalingGroup> asgList = createASGList();
+        String[] asgNames = {ASG1, ASG2};
+        AWSClient awsMock = createMockAWSClient(asgList, asgNames);
```

Comment:
I'm not following the logic for this test: an awsMock was created and setup but never used.  

It appears this test just looks to see if the owner tags are present in the test list rather then testing if the AWSClusterCrawler is behaving as expected.